src/App.jsx

```diff
@@ -1,10 +1,29 @@
+import { BrowserRouter } from 'react-router-dom';
 import React from 'react';
 import styled from 'styled-components';
+import NavBar from './NavBar';
 
-const Title = styled.h1`
-  color: blue;
+// Styling section
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+  background-color: #383854;
 `;
 
-const App = () => <Title>Hey Amir and Sami and Mario</Title>;
+// our functionality
```

Comment:
This comment and empty lines seem unnecessary as there is no code before the next comment